### ISSUE TITLE
Probably need a new source : XE is blocking this

### ISSUE THREAD

**Issue by moeadham** [hippasus/ExchangeRates#32]
Requests from XE.com are returning as:

<HTML>
<HEAD><TITLE>Autoextraction Prohibited</TITLE></HEAD>
<BODY>

**Comment by richardtan** [comment]
Yup, I forked the code and used https://currencylayer.com/ free API. It gives access to 1000 calls, so use use a cron job to poll it every hour and store all the currencies in the Google Datastore and use that to interface with the currencies. Hope that gives you some idea :)
